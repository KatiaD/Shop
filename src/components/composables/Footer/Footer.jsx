import React from "react";
import { FooterWrapper } from "./Footer.styled";

const displayName = "Footer";

const propTypes = {};

function Footer() {
  return (
    <FooterWrapper>
      <div>Onseo -2018</div>
    </FooterWrapper>
  );
}

Footer.displayName = displayName;
Footer.propTypes = propTypes;

export default Footer;
