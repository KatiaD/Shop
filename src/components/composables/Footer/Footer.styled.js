import styled from "styled-components";

export const FooterWrapper = styled.footer`
  display: flex;
  justify-content: center;
  align-content: space-between;
  padding: 30px;
  background: #ffa50054;
`;
