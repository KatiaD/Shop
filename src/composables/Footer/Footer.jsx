import React from 'react';

import FooterWrapper from './Footer.styled';

const displayName = 'Footer';

const propTypes = {};

const defaultProps = {};

function Footer() {
  return (
    <FooterWrapper>
         @Onseo-2018
    </FooterWrapper>
  );
}

Footer.displayName = displayName;
Footer.propTypes = propTypes;
Footer.defaultProps = defaultProps;

export default Footer;
