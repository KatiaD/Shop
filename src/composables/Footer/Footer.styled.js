import styled from 'styled-components';

const FooterWrapper = styled.footer`
  flex: 0 0 auto;
  background-color: ${({ theme: { colorDefault } }) => colorDefault};
  padding: 10px;
  color: ${({ theme: { colorAccent } }) => colorAccent};
  font: 400 12px/12px Ubuntu,sans-serif;
  text-align:center;
`;

export default FooterWrapper;
