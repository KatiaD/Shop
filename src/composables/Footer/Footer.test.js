import React from 'react';
import { shallow } from 'enzyme';

import Footer from './Footer';

describe('Given the Footer component', () => {
  describe('when the component is rendered', () => {
    let component;

    beforeEach(() => {
      component = shallow(<Footer/>);
    });

    it('should render the component with provided props', () => {
      expect(component).toMatchSnapshot();
    });
  });
});