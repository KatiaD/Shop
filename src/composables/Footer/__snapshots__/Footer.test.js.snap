// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given the Footer component when the component is rendered should render the component with provided props 1`] = `
<Footerstyled__FooterWrapper>
  @Onseo-2018
</Footerstyled__FooterWrapper>
`;
